### ISSUE TITLE
website down - DNS error (expired?)

### ISSUE THREAD

**Issue by techieshark** [imagemin/imagemin.io#1]
I went to http://imagemin.io/ but it seems to not be pointing to github's server anymore (for gh-pages):

![image](https://cloud.githubusercontent.com/assets/1072292/12072630/a68021b8-b14d-11e5-9246-115fbc0196ba.png)

**Comment by sindresorhus** [comment]
@kevva ?

**Comment by kevva** [comment]
Has it ever? I think I just bought it and did nothing with it.

**Comment by sindresorhus** [comment]
@kevva No. Can you set up a redirect to https://github.com/imagemin/imagemin?

**Comment by kevva** [comment]
I haven't renewed the domain name since I bought it. I probably will though when I start working on this again.
